docs/dr-health.md

```diff
@@ -0,0 +1,42 @@
+# Disaster Recovery
+
+DR health command is used to get connection status of one cluster from another cluster in mirroring enabled clusters. We fetch cephblockpool with mirrroing enabled if not found it will exit with relevant logs. Optionally, it accept all the args that are supported with `ceph status` command like `--debug-ms 1`.
```

Comment:
This paragraph and example needs to be moved inside the Health header.
```suggestion
The DR health command is used to get the connection status of one cluster from another cluster in mirroring-enabled clusters. The cephblockpool is queried with mirrroring enabled and If not found will exit with relevant logs. Optionally, args can be passed to be appended to `ceph status`, for example: `--debug-ms 1`.
```

Reply:
changed